packages/@aws-cdk/aws-apigatewayv2-authorizers/README.md

```diff
@@ -17,24 +18,120 @@
+
+The example below showcases default authorization, along with route authorization. It also shows how to remove authorization entirely for a route.
+
+- `GET /books` and `GET /books/{id}` use the default authorizer settings on the api
+- `POST /books` will require the [write:books] scope
+- `POST /login` removes the default authorizer (unauthenticated route)
+
+```ts
+const authorizer = new HttpJwtAuthorizer({
+  ...
+});
+
+const api = new HttpApi(stack, 'HttpApi', {
+  defaultAuthorizer: authorizer,
+  defaultAuthorizationScopes: ['read:books'],
+});
+
+api.addRoutes({
+  ...
+  path: '/books',
+  method: 'get',
+});
+
+api.addRoutes({
+  ...
+  path: '/books/{id}',
+  method: 'get',
+});
+
+api.addRoutes({
+  ...
+  path: '/books',
+  method: 'post',
+  authorizationScopes: ['write:books']
+});
+
+api.addRoutes({
+  ...
+  path: '/login',
+  method: 'post',
+  authorizer: new NoneAuthorizer(),
+});
+```
+
 ## JWT Authorizers
 
 JWT authorizers allow the use of JSON Web Tokens (JWTs) as part of [OpenID Connect](https://openid.net/specs/openid-connect-core-1_0.html) and [OAuth 2.0](https://oauth.net/2/) frameworks to allow and restrict clients from accessing HTTP APIs.
 
-When configured on a route, the API Gateway service validates the JWTs submitted by the client, and allows or denies access based on its content.
+When configured, API Gateway validates the JWT submitted by the client, and allows or denies access based on its content.
+
+The location of the token is defined by the `identitySource` which defaults to the http `Authorization` header. However it also [supports a number of other options](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-lambda-authorizer.html#http-api-lambda-authorizer.identity-sources). It then decodes the JWT and validates the signature and claims, against the options defined in the authorizer and route (scopes). For more information check the [JWT Authorizer documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-jwt-authorizer.html).
+
+Clients that fail authorization are presented with either 2 responses:
 
-API gateway uses the `identitySource` to determine where to look for the token. By default it checks the http `Authorization` header. However it also [supports a number of other options](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-lambda-authorizer.html#http-api-lambda-authorizer.identity-sources). It then decodes the JWT and validates the signature and claims, against the options defined in the authorizer and route (scopes). For more information check the [JWT Authorizer documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-jwt-authorizer.html).
+- `401 - Unauthorized` - When the JWT validation fails
+- `403 - Forbidden` - When the JWT validation is successful but the required scopes are not met
```

Comment:
This is true for all authorizers, no?

Should we move this up into the 'HTTP APIs' section?